.gitignore

```diff
@@ -16,6 +24,14 @@ HasalServer.log.*
 stage/*
 docs/*
 
+# File types to ignore
+*.zip
+*.diff
+*.json
```

Comment:
I'll suggest we specific what we'd like to have in the repo. Currently, every thing is dump to the Hasal folder and it make the code hard to maintain.